lib/operations/operation.js

```diff
@@ -1,11 +1,14 @@
 'use strict';
 
+const ReadPreference = require('../core/topologies/read_preference');
+
 const Aspect = {
   READ_OPERATION: Symbol('READ_OPERATION'),
   WRITE_OPERATION: Symbol('WRITE_OPERATION'),
   RETRYABLE: Symbol('RETRYABLE'),
   EXECUTE_WITH_SELECTION: Symbol('EXECUTE_WITH_SELECTION'),
-  NO_INHERIT_OPTIONS: Symbol('NO_INHERIT_OPTIONS')
+  NO_INHERIT_OPTIONS: Symbol('NO_INHERIT_OPTIONS'),
+  USE_PRIMARY_SERVER_SELECTION: Symbol('USE_PRIMARY_SERVER_SELECTION')
```

Comment:
Do we need a new aspect? In `4.0` we've already [solved this problem](https://github.com/mongodb/node-mongodb-native/blob/master/src/operations/command.ts#L71-L73) by fixing the `readPreference` to `primary` when constructing a `WRITE_OPERATION`.